conformance/Makefile.am

```diff
@@ -2,7 +2,11 @@
 
 conformance_protoc_inputs =                                    \
   conformance.proto                                            \
-  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto
+  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto 
+  
+conformance_proto2_protoc_inputs =                             \
```

Comment:
I tried to generate the proto file in one command, but it seems that it can't use proto2 file as input with "--ruby_out --objc_out --php_out" flag. There will be an error : 
"--ruby_out: google/protobuf/test_messages_proto2.proto: Can only generate Ruby code for proto3 .proto files."